package.json

```diff
@@ -19,33 +19,53 @@
     "rulesDirectory": {
       "type": "string",
       "title": "Custom rules directory (absolute path)",
-      "default": ""
+      "default": "",
+      "order": 2
```

Comment:
If you're going to add `order` to these, re-order them in `package.json` itself to match the `order` 😉.

Reply:
Hah, I don't know why I didn't just do that to begin with. I guess I was a bit shy contributing to a new repo and trying to keep my "lines modified" count down. 😝 It's fixed!